3rdparty/openexr/IlmImf/ImfAttribute.cpp

```diff
@@ -63,7 +63,7 @@ Attribute::~Attribute () {}
 
 namespace {
 
-struct NameCompare: std::binary_function <const char *, const char *, bool>
+struct NameCompare
```

Comment:
@asmorkalov Wrong commit is pushed

Reply:
My fault. I restored original commit, thanks to Github, it stores rewritten commits too.